sdk-extensions/metric-incubator/src/main/java/io/opentelemetry/sdk/viewconfig/ViewConfig.java

```diff
@@ -179,12 +186,47 @@ static View toView(ViewSpecification viewSpec) {
   }
 
   // Visible for testing
-  static Aggregation toAggregation(String aggregation) {
+  @SuppressWarnings("unchecked")
+  static Aggregation toAggregation(String aggregationName, Map<String, Object> aggregationArgs) {
+    Aggregation aggregation;
     try {
-      return AggregationUtil.forName(aggregation);
+      aggregation = AggregationUtil.forName(aggregationName);
     } catch (IllegalArgumentException e) {
       throw new ConfigurationException("Error creating aggregation", e);
     }
+    if (Aggregation.explicitBucketHistogram().equals(aggregation)) {
+      List<Double> bucketBoundaries =
+          Optional.ofNullable(
+                  ((List<Object>) getAsType(aggregationArgs, "bucket_boundaries", List.class)))
+              .map(
+                  objects ->
+                      objects.stream()
+                          .map(
+                              object1 -> {
```

Comment:
urg. Would even prefer just `o` to this parameter name. Could we extract the whole lambda into a method so we can name the parameter better?